node/package.json

```diff
@@ -7,7 +7,7 @@
     "@types/co-body": "^0.0.3",
     "@types/node": "12.x",
     "@types/ramda": "types/npm-ramda#dist",
-    "@vtex/api": "^3.62.4",
+    "@vtex/api": "6.13.0",
```

Comment:
Shouldn't you keep the caret to get bug fixes?

Reply:
It was automatic, now the version is the same from the runtime. Therefore I do not think we need the caret, but i will check with @tiagonapoli